hc/settings.py

```diff
@@ -120,7 +120,7 @@
 
 USE_TZ = True
 
-SITE_ROOT = "http://localhost:8000"
+SITE_ROOT = "http://healthchecks-vulcan.herokuapp.com"
```

Comment:
needs to be derived from the environment. Hard-coding this might yield unexpected results